### DISCUSSION TITLE
Is it possible to use .NET Aspire with Docker Engine via WSL command?

### DISCUSSION THREAD

**Discussion by devbrsa** [microsoft/aspire#6813] [Q&A] [answered]
Hi team,  

I'm exploring ways to use **.NET Aspire** without having Docker Desktop, Podman, or Rancher Desktop installed on Windows. Instead, I want to rely solely on the Docker Engine accessed through WSL commands, for example:  

```bash  
wsl docker run hello-world  
```  

Is there a way to configure .NET Aspire to work under such a setup? If not, are there any workarounds or plans to support this kind of setup in the future?  

Thanks for your time and support!

**Comment by davidfowl** [comment, marked as answer]
These are all the issues you'll run into https://github.com/dotnet/aspire/issues?q=is%3Aissue%20state%3Aopen%20label%3Awsl
